<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClothingItemsWindow</class>
 <widget class="QMainWindow" name="ClothingItemsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>891</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="categoriesLayout">
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #C1BBDF; 
    color: black; /* Text color */
    border: 1px solid rgb(0, 0, 0);
    font-family: 'Courier New';
    font-size: 14px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #AEA7DD; /* Slightly darker blue on hover */
}
QPushButton:pressed {
    background-color: #8F85D0; /* Even darker blue when pressed */
}
</string>
       </property>
       <property name="text">
        <string>COATS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #C1BBDF; 
    color: black; /* Text color */
    border: 1px solid rgb(0, 0, 0);
    font-family: 'Courier New';
    font-size: 14px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #AEA7DD; /* Slightly darker blue on hover */
}
QPushButton:pressed {
    background-color: #8F85D0; /* Even darker blue when pressed */
}
</string>
       </property>
       <property name="text">
        <string>TOPS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #C1BBDF; 
    color: black; /* Text color */
    border: 1px solid rgb(0, 0, 0);
    font-family: 'Courier New';
    font-size: 14px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #AEA7DD; /* Slightly darker blue on hover */
}
QPushButton:pressed {
    background-color: #8F85D0; /* Even darker blue when pressed */
}
</string>
       </property>
       <property name="text">
        <string>BOTTOMS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #C1BBDF; 
    color: black; /* Text color */
    border: 1px solid rgb(0, 0, 0);
    font-family: 'Courier New';
    font-size: 14px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #AEA7DD; /* Slightly darker blue on hover */
}
QPushButton:pressed {
    background-color: #8F85D0; /* Even darker blue when pressed */
}
</string>
       </property>
       <property name="text">
        <string>SHOES</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>891</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="clothesLayout">
     <item row="0" column="0">
      <widget class="QScrollArea" name="scrollArea">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>887</width>
          <height>507</height>
         </rect>
        </property>
        <widget class="QFrame" name="frame">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>170</width>
           <height>170</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
        </widget>
        <widget class="QFrame" name="frame_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>240</y>
           <width>170</width>
           <height>170</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
        </widget>
        <widget class="QFrame" name="frame_3">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>30</y>
           <width>170</width>
           <height>170</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
        </widget>
        <widget class="QFrame" name="frame_4">
         <property name="geometry">
          <rect>
           <x>460</x>
           <y>30</y>
           <width>170</width>
           <height>170</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
        </widget>
        <widget class="QFrame" name="frame_5">
         <property name="geometry">
          <rect>
           <x>680</x>
           <y>30</y>
           <width>170</width>
           <height>170</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
        </widget>
        <widget class="QFrame" name="frame_6">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>240</y>
           <width>170</width>
           <height>170</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
        </widget>
        <widget class="QFrame" name="frame_7">
         <property name="geometry">
          <rect>
           <x>460</x>
           <y>240</y>
           <width>170</width>
           <height>170</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
        </widget>
        <widget class="QFrame" name="frame_8">
         <property name="geometry">
          <rect>
           <x>680</x>
           <y>240</y>
           <width>170</width>
           <height>170</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
        </widget>
        <widget class="QFrame" name="frame_9">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>450</y>
           <width>170</width>
           <height>170</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
        </widget>
        <widget class="QFrame" name="frame_10">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>450</y>
           <width>170</width>
           <height>170</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
        </widget>
        <widget class="QFrame" name="frame_11">
         <property name="geometry">
          <rect>
           <x>460</x>
           <y>450</y>
           <width>170</width>
           <height>170</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
        </widget>
        <widget class="QFrame" name="frame_12">
         <property name="geometry">
          <rect>
           <x>680</x>
           <y>450</y>
           <width>170</width>
           <height>170</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="pageLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
    font-family: 'Courier New';
    font-size: 30px;
    font-weight: bold;
 border : 1px solid black;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>All Clothing Items</string>
    </property>
   </widget>
   <widget class="QPushButton" name="viewCloset_button">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #C1BBDF; 
    color: black; /* Text color */
    border: 1px solid rgb(0, 0, 0);
    font-family: 'Courier New';
    font-size: 14px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #AEA7DD; /* Slightly darker blue on hover */
}
QPushButton:pressed {
    background-color: #8F85D0; /* Even darker blue when pressed */
}
</string>
    </property>
    <property name="text">
     <string>View Closet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="viewClothingItems_button">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #C1BBDF; 
    color: black; /* Text color */
    border: 1px solid rgb(0, 0, 0);
    font-family: 'Courier New';
    font-size: 14px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #AEA7DD; /* Slightly darker blue on hover */
}
QPushButton:pressed {
    background-color: #8F85D0; /* Even darker blue when pressed */
}
</string>
    </property>
    <property name="text">
     <string>View Saved Outfits</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backToMainMeny_button">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #C1BBDF; 
    color: black; /* Text color */
    border: 1px solid rgb(0, 0, 0);
    font-family: 'Courier New';
    font-size: 14px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #AEA7DD; /* Slightly darker blue on hover */
}
QPushButton:pressed {
    background-color: #8F85D0; /* Even darker blue when pressed */
}
</string>
    </property>
    <property name="text">
     <string>Back to Main Menu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uploadItem_button">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>90</y>
      <width>141</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #C1BBDF; 
    color: black; /* Text color */
    border: 1px solid rgb(0, 0, 0);
    font-family: 'Courier New';
    font-size: 14px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #AEA7DD; /* Slightly darker blue on hover */
}
QPushButton:pressed {
    background-color: #8F85D0; /* Even darker blue when pressed */
}
</string>
    </property>
    <property name="text">
     <string>Upload Item</string>
    </property>
   </widget>
   <widget class="QPushButton" name="massDeleteButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>30</y>
      <width>141</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #C1BBDF; 
    color: black; /* Text color */
    border: 1px solid rgb(0, 0, 0);
    font-family: 'Courier New';
    font-size: 14px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #AEA7DD; /* Slightly darker blue on hover */
}
QPushButton:pressed {
    background-color: #8F85D0; /* Even darker blue when pressed */
}
</string>
    </property>
    <property name="text">
     <string>Delete Item/s</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_13">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>1021</width>
      <height>761</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/backgrounds.png);
background-repeat: no-repeat;
background-position: center;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
   </widget>
   <zorder>frame_13</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>pageLabel</zorder>
   <zorder>viewCloset_button</zorder>
   <zorder>viewClothingItems_button</zorder>
   <zorder>backToMainMeny_button</zorder>
   <zorder>uploadItem_button</zorder>
   <zorder>massDeleteButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
